Microsoft.Toolkit.Mvvm/Microsoft.Toolkit.Mvvm.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
```

Comment:
This has no implications for UWP consumers, eh?

Reply:
Nope, in fact this is the same `LangVersion` that the main `Microsoft.Toolkit` package is using since #3131 got merged. We're not using any C# 8 features that require runtime support (eg. ranges or default interfaces methods), but just things that are purely syntactic sugar (eg. `??=` operator, nullable reference types, etc.). UWP is fully supported, in fact the tests for this package are running on UWP too in this branch.